modules/openapi-generator/src/main/resources/aspnetcore/2.0/controller.mustache

```diff
@@ -1,8 +1,10 @@
 {{>partial_header}}
 using System;
 using System.Collections.Generic;
-using Microsoft.AspNetCore.Mvc;{{#useSwashbuckle}}
-using Swashbuckle.AspNetCore.SwaggerGen;{{/useSwashbuckle}}
+using Microsoft.AspNetCore.Mvc;
+{{#useSwashbuckle}}
```

Comment:
This adds a newline, which I don't think is intentional.

Reply:
I think the version of jmustache has better handling of newline: https://github.com/samskivert/jmustache#newline-trimming